Error with LBR IIWA: robot is not connected

I am trying to setup your packages to work with Kuka LBR IIWA. I am using ubuntu 14.04 LTS and ros Indigo. I found the robot connected with an wireless setting. Therefore, I didn't change the configuration. I uploaded the sunrise project to controller with no error or red sign in IDE. I followed roscore side. Next I ping with cabinet address and got reply. 
However, when I run the lunch file using  roslaunch iiwa_tool_examples iiwa_tool_read.launch, I found the error, [ERROR] [1458371985.836149274]: Robot is not connected.....  
 
afroza@afroza-XPS-8700:~/ros_ws$ roslaunch iiwa_tool_examples iiwa_tool_read.launch
... logging to /home/afroza/.ros/log/8b1ff4da-eda1-11e5-8d62-989096a52895/roslaunch-afroza-XPS-8700-6122.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.
started roslaunch server http://afroza-XPS-8700:41750/
SUMMARY
========
PARAMETERS
 * /iiwa/ReadRobotState/joint_position_topic: /iiwa/state/Joint...
 * /iiwa/ReadRobotState/ros_rate: 1000
 * /iiwa/publishJointStates: True
 * /iiwa/robot_description: <?xml version="1....
 * /iiwa/robot_description_kinematics/manipulator/kinematics_solver: kdl_kinematics_pl...
 * /iiwa/robot_description_kinematics/manipulator/kinematics_solver_attempts: 3
 * /iiwa/robot_description_kinematics/manipulator/kinematics_solver_search_resolution: 0.005
 * /iiwa/robot_description_kinematics/manipulator/kinematics_solver_timeout: 0.005
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_1/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_1/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_1/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_1/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_2/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_2/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_2/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_2/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_3/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_3/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_3/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_3/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_4/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_4/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_4/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_4/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_5/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_5/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_5/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_5/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_6/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_6/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_6/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_6/max_velocity: 10
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_7/has_acceleration_limits: False
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_7/has_velocity_limits: True
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_7/max_acceleration: 0
 * /iiwa/robot_description_planning/joint_limits/iiwa_joint_7/max_velocity: 10
 * /iiwa/robot_description_semantic: <?xml version="1....
 * /iiwa/toolName: tool
 * /rosdistro: indigo
 * /rosversion: 1.11.16
NODES
  /iiwa/
    ReadRobotState (iiwa_tool_examples/read_robot_state)
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    rviz_afroza_XPS_8700_6122_4091623036572324931 (rviz/rviz)
ROS_MASTER_URI=http://172.31.1.2:11311
core service [/rosout] found
process[iiwa/robot_state_publisher-1]: started with pid [6143]
process[iiwa/ReadRobotState-2]: started with pid [6144]
process[iiwa/rviz_afroza_XPS_8700_6122_4091623036572324931-3]: started with pid [6145]
[ERROR] [1458371905.833199520]: Robot is not connected...
[ INFO] [1458371905.886202985]: rviz version 1.11.10
[ INFO] [1458371905.886257469]: compiled against OGRE version 1.8.1 (Byatis)
[ INFO] [1458371906.552963147]: Stereo is NOT SUPPORTED
[ INFO] [1458371906.553024826]: OpenGl version: 4.5 (GLSL 4.5).
[ INFO] [1458371907.212131210]: Loading robot model 'iiwa7_tool'...
[ INFO] [1458371907.212188297]: No root joint specified. Assuming fixed joint
[ERROR] [1458371910.833342426]: Robot is not connected...
-------------
Please help me in this regard.
Thank you

--- a/iiwa_ros_java/src/de/tum/in/camp/kuka/ros/config.txt
+++ b/iiwa_ros_java/src/de/tum/in/camp/kuka/ros/config.txt
@@ -1,4 +1,4 @@
 robot_name: iiwa
 master_ip: 160.69.69.100
 master_port: 11311
-ntp_with_host: true
+ntp_with_host: false

